src/core/MousePointer.gd

```diff
@@ -89,9 +89,7 @@ func determine_global_mouse_pos() -> Vector2:
 	# scaling the game in full-creen.
 	if get_viewport() and get_viewport().has_node("Camera2D"):
 		zoom = get_viewport().get_node("Camera2D").zoom
-	var offset_mouse_position = \
-			get_tree().current_scene.get_global_mouse_position() \
-			- get_viewport_transform().origin
+	var offset_mouse_position = self.get_global_mouse_position() - get_viewport_transform().origin
```

Comment:
i think i may need to revert this.

Reply:
so the .current_scene does exist in CLI?